code/modules/admin/verbs/randomverbs.dm

```diff
@@ -484,7 +484,7 @@ Traitors and the like can also be revived with the previous role mostly intact.
 	if(!M.adminfrozen)
 		var/sleepies = M.AmountSleeping()
 		M.adminfrozen = sleepies ? sleepies : 1
-		M.SetSleeping(20000)//2k seconds to get your admin shit together
+		M.SetSleeping(200000)//20k seconds to get your admin shit together

```

Comment:
Why, I mean, fine, but 2k was fine... 2k is literally a bit more than 30 mins.... 20k is 5 and a half hours of frozen time.